### PR TITLE
Update author details

### PR DESCRIPTION
**Pull request by krstnfx**
### Motivation:

Update author details for Kristina Fox and Holly Borla

### Modifications:

Add Gravatar images and updated bios

### Result:

Screen Shot 2022-06-17 at 11 30 12 AM

### CODE REVIEW

**Comment by shahmishal** [comment]
CC @hborla

**Review by hborla — APPROVED** [review]
Thank you!

**Review by alexandersandberg — APPROVED** [review]
We are [planning](https://forums.swift.org/t/workgroup-vs-work-group/57107/18) to introduce a consistent style of spelling "workgroup(s)" without a space. This is the recommended spelling by the Apple Style Guide—which we probably will try to follow more closely going forward (more on that later).

While author bios can and should be personal, and likely won't strictly follow the same guidelines, I just wanted to make you aware of this. 🙂

**Comment by alexandersandberg** [comment]
@swift-ci test

File: _data/authors.yml [hunk 1]
```diff
@@ -230,13 +230,15 @@ hborla:
    email: hborla@apple.com
    github: hborla
    twitter: hollyborla
-   about: "Holly Borla is an engineer on the Swift compiler team at Apple, and a member of the Diversity in Swift work group."
+   gravatar: b3e284dd808123c26f476d75da0b5b3c
+   about: "Holly Borla is an engineer on the Apple Swift team, and a member of the Swift Core Team, Language work group, and Diversity in Swift work group."
```
Comment (shahmishal) [inline _data/authors.yml:234]:
```suggestion
   about: "Holly Borla is an engineer on the Apple Swift team, and a member of the Swift Core Team, Language work group, and Diversity in Swift workgroup."
```

File: _data/authors.yml [hunk 2]
```diff
@@ -230,13 +230,15 @@ hborla:
...
 
 krstnfx:
    name: Kristina Fox
    github: krstnfx
    twitter: krstnfx
-   about: "Kristina Fox is an iOS engineer on the health team at Apple, and a member of the Diversity in Swift work group."
+   gravatar: 5e142f580e63beae5db6a7ba25e9162c
+   about: "Kristina Fox is an iOS engineering manager on the health team at Apple, and a member of the Diversity in Swift and Swift.org website work groups."
```
Comment (shahmishal) [inline _data/authors.yml:241]:
```suggestion
   about: "Kristina Fox is an iOS engineering manager on the health team at Apple, and a member of the Diversity in Swift and Swift.org website workgroups."
```

File: _data/authors.yml [hunk 3]
```diff
@@ -230,13 +230,15 @@ hborla:
    email: hborla@apple.com
    github: hborla
    twitter: hollyborla
-   about: "Holly Borla is an engineer on the Swift compiler team at Apple, and a member of the Diversity in Swift work group."
+   gravatar: b3e284dd808123c26f476d75da0b5b3c
+   about: "Holly Borla is an engineer on the Apple Swift team, and a member of the Swift Core Team, Language work group, and Diversity in Swift workgroup."
```
Comment (alexandersandberg) [inline _data/authors.yml:234]:
```suggestion
   about: "Holly Borla is an engineer on the Apple Swift team, and a member of the Swift Core Team, Language workgroup, and Diversity in Swift workgroup."
```

🙂

**Comment by alexandersandberg** [comment]
@swift-ci test

**Review by alexandersandberg — APPROVED** [review]

**Pull request merged**